### ISSUE TITLE
Version typed-ast==1.4.0 (released June 5th) binaries are not available for py34

### ISSUE THREAD

**Issue by jgsogo** [PyCQA/astroid#676]
### Steps to reproduce
1. Machine without Microsoft Windows SDK 7.1
2. `pip install astroid` (using python 3.4, it will trigger a build)
3. Error in [output](https://conan-ci.jfrog.info/blue/organizations/jenkins/ConanNightly/detail/ConanNightly/518/pipeline):

```
  Running setup.py install for typed-ast: started
    Running setup.py install for typed-ast: finished with status 'error'
    ERROR: Complete output from command 'd:\j\t\9f6d\py34\disabled_revisions\venv\scripts\python.exe' -u -c 'import setuptools, tokenize;__file__='"'"'C:\\Users\\Jenkins\\AppData\\Local\\Temp\\2\\pip-install-2jlu6nis\\typed-ast\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Jenkins\AppData\Local\Temp\2\pip-record-r0fif5f_\install-record.txt' --single-version-externally-managed --compile --install-headers 'd:\j\t\9f6d\py34\disabled_revisions\venv\include\site\python3.4\typed-ast':
    ERROR: running install
    running build
    running build_py
    creating build
    creating build\lib.win32-3.4
    creating build\lib.win32-3.4\typed_ast
    copying typed_ast\ast27.py -> build\lib.win32-3.4\typed_ast
    copying typed_ast\ast3.py -> build\lib.win32-3.4\typed_ast
    copying typed_ast\conversions.py -> build\lib.win32-3.4\typed_ast
    copying typed_ast\__init__.py -> build\lib.win32-3.4\typed_ast
    package init file 'ast3\tests\__init__.py' not found (or not a regular file)
    creating build\lib.win32-3.4\typed_ast\tests
    copying ast3\tests\test_basics.py -> build\lib.win32-3.4\typed_ast\tests
    running build_ext
    building '_ast27' extension
    error: Microsoft Visual C++ 10.0 is required. Get it with "Microsoft Windows SDK 7.1": www.microsoft.com/download/details.aspx?id=8279
```
### Current behavior
`astroid` fails to install (fails to build) as a consequence of this new `typed-ast` version that requires a Microsoft Windows SDK to compile.

### Expected behavior
Install `astroid` successfully

### Proposed solution:
Freeze `typed-ast` version to be `<1.4`
Reported to `typed-ast` repo to check that this change was intentional: https://github.com/python/typed_ast/issues/122

### Astroid version:
Wheel: `astroid-2.2.5-py3-none-any.whl`

**Comment by thisch** [comment]
Note that support for python3.4 has been dropped from master in https://github.com/PyCQA/astroid/commit/4a4b7faf9de8437f2b96bc7285802fd1230b7a9a

> Freeze typed-ast version to be <1.4
typed-ast is already freezed in tox.ini, but unfortunately not in `__pkginfo__.py`

**Comment by PCManticore** [comment]
Yeah, we should freeze it in `__pkginfo__` as well.

**Comment by thisch** [comment]
Thx for the report @jgsogo. I've created a fix in #683, but since support for python3.4 has been removed from astroid-master you have to upgrade to at least python 3.5 to test it. 
Probably it makes sense to backport this fix to astroid 2.2.
